docs/reference/api/rtos/rtos.md

```diff
@@ -1,14 +1,16 @@
 <h2 id="rtos-api">RTOS overview</h2>
 
-The Arm Mbed RTOS is a C++ wrapper over the Keil RTX code. For more information about Keil RTX, check [the Keil CMSIS-RTOS tutorial](https://github.com/ARM-software/CMSIS/raw/master/CMSIS/Documentation/RTX/CMSIS_RTOS_Tutorial.pdf) and [the element14 introduction to Keil RTX](https://www.element14.com/community/docs/DOC-46650/l/arm-keil-rtx-real-time-operating-system-overview). You can use these resources as a general introduction to RTOS principles; it is important to be familiar with the concepts behind an RTOS in order to understand this guide.
+Arm Mbed OS provides C++ APIs to manage RTOS objects such as thread, synchronization objects and timer. It also provides interfaces for attaching an application-specific idle hook function and to read the tick count from OS. The Arm Mbed RTOS layer also handles RTOS errors and reports them into the Mbed OS error handling system.
```

Comment:
> Arm Mbed OS provides C++ APIs to manage

is the same as saying The Arm Mbed RTOS is a C++ wrapper over the Keil RTX code just in a different way. How about something like along the lines of the mbed os rtos capabilities include things such as ...